src/store/modules/authModule.js

```diff
@@ -0,0 +1,27 @@
+
+import { authAPI } from "@/api/authAPI";
+
+export const authModule = {
+    namespaced: true,
+
+    state: {
+        user: null,
+      },
+      mutations: {
+        SET_USER_DATA (state, userData) {
+          state.user = userData;
+          localStorage.setItem('user', JSON.stringify(userData));
+        },
+      },
+      actions: {
+          async onLogin({ commit }, credentials) {
+              try {
+                  const res = await authAPI.login(credentials);
```

Comment:
const { data } =

Reply:
// const isAuthorized = () => localStorage.getItem('user');
// const authGuard = (to, from, next) => {
//   if(!isAuthorized) next({ name: '/' });
//   else next();
// }

const routes = [
  {
    path: "/",
    name: "login",
    component: Login,
  },
  // {
  //   path: "/order-list",
  //   name: "order-list",
  //   component: OrderList,
  //   beforeEnter: authGuard
  // },
  {
    path: "/order-list",
    name: "order-list",
    component: () => import('@/views/order-list.vue'),
    meta: { needAuth: true }
  },
];